packages/block-library/src/embed/util.js

```diff
@@ -114,6 +124,9 @@ export const createUpgradedEmbedBlock = (
 			! providerNameSlug );
 	if ( shouldCreateNewBlock ) {
 		return createBlock( DEFAULT_EMBED_BLOCK, {
+			align,
```

Comment:
Sorry for being late here, but doesn't it make sense to just add all attributes here?

Reply:
Yeah, I think we could include all of them, my initial concern was if the extra ones would produce any kind of side-effect:
- `className`
- `previewable`
- `responsive`

So far what I tested looks safe so I'm going to apply your suggestion, thanks @ntsekouras 🙇 !

Reply:
> So far what I tested looks safe so I'm going to apply your suggestion, thanks @ntsekouras 🙇 !

I included this suggestion in [this commit](https://github.com/WordPress/gutenberg/pull/33235/commits/f9c347f369761c7b084e95dc5a18786f09a0bce4).